tests/Internal/Codebase/InternalCallMapHandlerTest.php

```diff
@@ -446,36 +617,42 @@ private function isIgnored(string $functionName): bool
      */
     public function testIgnoredFunctionsStillFail(string $functionName, array $callMapEntry): void
     {
-        if (!$this->isIgnored($functionName)) {
+        $functionIgnored = $this->isIgnored($functionName);
+        if (!$functionIgnored && !$this->isReturnTypeOnlyIgnored($functionName)) {
             // Dummy assertion to mark it as passed
             $this->assertTrue(true);
             return;
         }
 
-        $this->expectException(ExpectationFailedException::class);
+        $function = new ReflectionFunction($functionName);
+        /** @var string $entryReturnType */
+        $entryReturnType = array_shift($callMapEntry);
+
+        if ($functionIgnored) {
+            try {
+                /** @var array<string, string> $callMapEntry */
```

Comment:
can you document this with an @param maybe?

Reply:
Added one for the CallMap entry. I don't think there's a way to define the shape so it's just `array<int|string, string>`

I am traveling until Friday so you can make changes or wait until I'm back for further issues.